lib/addons/secrets-store/csi-driver-provider-aws-secrets.ts

```diff
@@ -44,6 +44,11 @@ export interface KubernetesSecret {
      */
     secretName: string;
 
+    /**
+     * An alias for the AWS secret once it is synced as a Kubernetes secret.
+     */
+    secretAlias?: string;
```

Comment:
is it mutually exclusive with jmes path? if so let's reflect, it will be in the generated docs.

Reply:
The docs for the AWS provider do not mention any mutual exclusivity: https://github.com/aws/secrets-store-csi-driver-provider-aws/blob/main/README.md?plain=1#L118

The property only appears to be called during file setup for the mounted secret:
https://github.com/aws/secrets-store-csi-driver-provider-aws/blob/109d74a2aed9e7e3fe5227c0de14d5d2b9962961/provider/secret_descriptor.go#L96-L99